webpack.config.prod.js

```diff
@@ -16,10 +58,10 @@ module.exports = {
     'react-datepicker': 'react-datepicker',
   },
   output: {
-    path: path.resolve(__dirname, 'dist'),
-    filename: 'index.js',
-    library: 'Plasma',
-    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'lib'),
+    filename: '[name].js',
+    library: '',
+    libraryTarget: 'commonjs2',
```

Comment:
So two questions:
1. Why do we have a different config for `prod` and `dist`? Why the different name?
1. Why does one use `umd` and the other `commonjs2`?

Reply:
One is for a distributable file and the other is for a code split tree shaken file.

Eventually, when we remove react-select as a real dependency and don't need to bundle it in. we can just sue `babel src -d lib` and we will not need webpack at all for the tree shaken version.

Reply:
Seems like `umd` covers `commonjs`, so why both?
How about `esm` (if supported)?

Reply:
Possibly.

This is what works right now. 

The dist file still can only be *one* file, the module can be multiple using require, so that is what we are doing here -- we are using commonjs in spacestation, there is no need for umd

Reply:
@GeReV we can get to esm and regular babel building when we remove react-select@2 as a direct dependency